src/pages/library/dataBrowser-utils.js

```diff
@@ -56,6 +56,12 @@ export const isDatarepoSnapshot = dataset => {
   return _.toLower(dataset['dcat:accessURL']).includes(datarepoSnapshotUrlFragment)
 }
 
+export const renderConsortium = dataset => _.flow(
+  _.map(dataCollection => dataCollection['dct:title']),
+  _.join(', ')
+)(dataset['TerraDCAT_ap:hasDataCollection'])
```

Comment:
Maybe this could be refactored a bit more now. It seems like there should be a way to combine this code with the code in `Databrowser.js`.

Reply:
I bounced around a few ideas in my head but the tldr is that I think, yeah, there probably is. I will look into it.